### ISSUE TITLE
Proof of concept guidelines?

### ISSUE THREAD

**Issue by reggi** [reggi/oneapp#1]
One HTML / JavaScript file that can be converted / compiled into
- [ ] Android
- [ ] iPhone
- [ ] Windows Phone
- [ ] Blackberry
- [ ] Firefox OS
- [ ] Web (i.e. AppCache)
- [ ] OSX
- [ ] Windows
- [ ] Linux
- [ ] Chrome OS

(feel free to add or subtract to add to this list or just ~~strikeout~~)

using some combination of 
- [nwjs/nw.js](https://github.com/nwjs/nw.js) (formerly know as node-webkit)
- [atom/atom-shell](https://github.com/atom/atom-shell)
- [chrome apps](https://developer.chrome.com/apps/about_apps)
- [phonegap/phonegap](https://github.com/phonegap/phonegap)
- [apache/cordova-cli](https://github.com/apache/cordova-cli/)

cc @nolanlawson

**Comment by nolanlawson** [comment]
added FXOS, web, chrome apps ^

**Comment by nolanlawson** [comment]
Also it should probably handle AppCache in some automated way; the [appcache-nanny](https://github.com/gr2m/appcache-nanny) seems pretty good for that.

**Comment by dpogue** [comment]
ServiceWorkers? They're supposed to be the way going forward for offline apps and caching.

**Comment by nolanlawson** [comment]
True, but [they're not very well-supported yet](https://jakearchibald.github.io/isserviceworkerready/).
